lib/IDE/SyntaxModel.cpp

```diff
@@ -1318,6 +1315,7 @@ bool ModelASTWalker::findUrlStartingLoc(StringRef Text,
       return true;
     }
   }
+#endif
   return false;
```

Comment:
It looks like the 'return false' is reachable with and without SWIFT_HAVE_WORKING_STD_REGEX?